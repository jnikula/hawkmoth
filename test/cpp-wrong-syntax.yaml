domain: c
directive-arguments:
  - cpp-wrong-syntax.h
example-title: C++ syntax in C source
example-priority: 1
