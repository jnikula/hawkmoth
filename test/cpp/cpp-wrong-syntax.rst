
.. c:var:: int foo

   C++ class inside a '.h'.

