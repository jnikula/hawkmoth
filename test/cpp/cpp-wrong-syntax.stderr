ERROR: cpp-wrong-syntax.h:2: unknown type name 'class'
ERROR: cpp-wrong-syntax.h:2: expected ';' after top level declarator
